pandas-stubs/_libs/tslibs/timestamps.pyi

```diff
@@ -111,17 +128,19 @@ class Timestamp(datetime):
     def date(self) -> _date: ...
     def time(self) -> _time: ...
     def timetz(self) -> _time: ...
-    def replace(
+    # Override since fold is more precise than datetime.replace(fold:int)
+    # Violation of Liskov substitution principle
```

Comment:
Not sure if it is because of `fold` being `Literal` or because we use `int | None` for most of the arguments, or because in typeshed, there is a `*` before `fold` .  Can you check and update the comment appropriately?

Reply:
It is because the super method is only int while here it is Literal[0,1]

Reply:
There is no * before fold in pandas main

Reply:
Comment updated.